apps/BasicExample/src/App.tsx

```diff
@@ -49,7 +49,7 @@ Stack.setRoutes({
 
 export default function App() {
   return (
-    <GestureHandlerRootView style={{ flex: 1 }}>
+    <GestureHandlerRootView style={{ flex: 1, paddingTop: 50 }}>
```

Comment:
Can this be on `SafeAreaView` (and only on Android) instead?

Reply:
Yup, changed in [3788cdd](https://github.com/software-mansion/react-native-gesture-handler/pull/3490/commits/3788cdd5096de08d7cd89edda23c342a38b16e59)